server/routes/form.js

```diff
@@ -121,12 +121,19 @@ module.exports = function Index({
     '/categoriser/provisionalCategory/:bookingId',
     asyncMiddleware(async (req, res, transactionalDbClient) => {
       const section = 'categoriser'
-      const form = 'provisionalCategory'
+      const user = await userService.getUser(res.locals)
+      res.locals.user = { ...user, ...res.locals.user }
+      const isFemale = res.locals.user.activeCaseLoad.female
+      const form = isFemale ? 'womensProvisionalCategory' : 'provisionalCategory'
       const { bookingId } = req.params
       const result = await buildFormData(res, req, section, form, bookingId, transactionalDbClient)
 
       if (result.data.openConditionsRequested) {
         res.redirect(`/form/openConditions/provisionalCategory/${bookingId}`)
+      } else if (isFemale) {
+        const suggestedCat = formService.computeFemaleSuggestedCat(result.data)
```

Comment:
Based on the clarification on today's meeting we don't need to calculate anything here, suggested cat will be closed.